omero/sysadmins/unix/install-web/web-deployment.txt

```diff
@@ -294,8 +257,8 @@ using :property:`omero.web.wsgi_args`:
 EXPERIMENTAL: Gunicorn advanced configuration
 ---------------------------------------------
 
-.. note:: Experimental configurations are not ready for production use.
-    Configuration may change. Features may not work properly.
+.. note:: The following configuration options have yet to be extensively tested in a
+          production environment, use at your own risk.
```

Comment:
Unnecessary extra whitespace here, you don't need to indent the 2nd line this far, just to underneath the 'o' of note